test/test_transformers.py

```diff
@@ -1118,7 +1123,39 @@ def test_efficient_attention_gradcheck(self, contiguous_inputs: bool):
         # in fp32
         assert gradcheck(lambda *args, **kwargs:
                          wrapper_set_seed(torch.ops.aten._efficient_attention_forward, *args, **kwargs),
-                         (query, key, value, None, None, None, True, False), fast_mode=True, atol=8e-5, rtol=1e-3)
+                         (query, key, value, None, None, None, True, False), fast_mode=False, atol=8e-5, rtol=1e-3)
```

Comment:
@albanD This test if passing with fast_mode = True. The new test I wrote below uses this torch.ops.aten function in it's backward. Those tests also fail in slow mode but pass with fast_mode = True. The kerenls don't have a fp64 implementation that is why it is done in torch.float32. 

All this context to say, from reading the fast_mode=True in the docs it doesn't say one mode is more accurate than the other. However I would assume slow is more accurate. How would I proceed in verifying this computed grad is correct?

@cpuhrsch @danthe3rd

Reply:
I think noteworthy is that the "math" kernel also fails with fast_mode=False in fp32 and these tolerances

Reply:
A case where fast mode may fail to catch incorrect gradients is the case when a few elements of the Jacobian are wrong, but the rest is correct. (In contrast, if we had something that was element-wise, fast mode would very likely catch this failure).

If you're only able to run in fp32, comparing the size of the error to what you get when running forward as a composite operator is a good proxy though.